tests/Mocks/DatabasePlatformMock.php

```diff
@@ -148,10 +133,15 @@ protected function initializeDoctrineTypeMappings()
     /**
      * @param array $field
      * @return string|void
-     * @throws DBALException
+     * @throws Exception
      */
     public function getBlobTypeDeclarationSQL(array $field)
     {
-        throw DBALException::notSupported(__METHOD__);
+        throw Exception::notSupported(__METHOD__);
+    }
+
+    public function getCurrentDatabaseExpression(): string {
+        throw Exception::notSupported(__METHOD__);
+        // TODO: Implement getCurrentDatabaseExpression() method.
```

Comment:
Is this method needed for the mock? If so, then the todo needs an implementation.
Otherwise, this todo can be deleted.

Reply:
Todo deleted